index.html

```diff
@@ -506,15 +506,5 @@ <h2>Projects and Companies using Ember Simple Auth</h2>
       </div>
     </div>
     <script src="https://yandex.st/highlightjs/7.4/highlight.min.js"></script>
-    <script type="text/javascript">
-      hljs.initHighlightingOnLoad();
```

Comment:
do we still need highlightjs here?

Reply:
I think it's fine to remove this but should be done in a separate PR.